src/TraceEvent/Symbols/NativeSymbolModule.cs

```diff
@@ -1175,12 +1203,173 @@ public byte[] GetTypeMDTokenMap()
         private ManagedSymbolModule m_managedPdb;
         private bool m_managedPdbAttempted;
 
-        internal SymbolReader m_reader;
-        internal IDiaSession m_session;
-        private IDiaDataSource3 m_source;
-        private IDiaEnumSymbolsByAddr m_symbolsByAddr;
+        internal readonly IDiaSession m_session;
+        private readonly SymbolReader m_reader;
+        private readonly IDiaDataSource3 m_source;
+        private readonly IDiaEnumSymbolsByAddr m_symbolsByAddr;
+        private readonly Lazy<IReadOnlyDictionary<uint, string>> m_heapAllocationSites; // rva => typename
 
         #endregion
+
+        /// <summary>
```

Comment:
Please put this in the private region before the private fields (I want fields to be last in the class so they are easy to find).